core/src/main/java/tc/oc/pgm/command/TeamCommand.java

```diff
@@ -48,8 +48,8 @@ public void force(
             "join.ok.force",
             TextColor.GRAY,
             joiner.getName(NameStyle.FANCY),
-            joiner.getParty().getName(NameStyle.FANCY),
-            oldParty.getName(NameStyle.FANCY)));
+            joiner.getParty().getComponentName(),
+            oldParty.getComponentName()));
```

Comment:
Not sure where we define `Party#getComponentName`, but it should just be `Party#getName`